.github/workflows/main.yml

```diff
@@ -0,0 +1,26 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+    # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+    - uses: actions/checkout@v2
+
+    - name: GitHub Action for pylint
+      uses: cclauss/GitHub-Action-for-pylint@0.7.0
```

Comment:
It's preferred to define the same version of python that the application supports to run pylint.

It can be achieved adding this option:

```
with:
    python-version: '3.x' 
```

Reply:
Also, I think you need to define 

```
with:
  args = "pip install -r requirements.txt ; pylint gdrive
```

in order to run pylint using this cclauss/GitHub-Action-for-pylint@0.7.0 per documentation in: https://github.com/cclauss/GitHub-Action-for-pylint it only run pylint on files with .py extension and the gdrive script does not have .py extension.